nopol_examples.nopol_example_8.NopolExampleTest#test_1
.
Time: 0.082

OK (1 test)

nopol_examples.nopol_example_8.NopolExampleTest#test_2
.E
Time: 0.007
There was 1 failure:
1) test_2(nopol_examples.nopol_example_8.NopolExampleTest)
java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at nopol_examples.nopol_example_8.NopolExampleTest.test_2(NopolExampleTest.java:17)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:47)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:44)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:271)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:70)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:50)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:238)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:63)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:236)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:53)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:229)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:309)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:160)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at fr.inria.lille.repair.MethodTestRunner.runTest(MethodTestRunner.java:44)
	at fr.inria.lille.repair.MethodTestRunner.access$000(MethodTestRunner.java:12)
	at fr.inria.lille.repair.MethodTestRunner$1.call(MethodTestRunner.java:20)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)

FAILURES!!!
Tests run: 1,  Failures: 1

nopol_examples.nopol_example_8.NopolExampleTest#test_3
.
Time: 0.004

OK (1 test)

nopol_examples.nopol_example_8.NopolExampleTest#test_4
.
Time: 0.002

OK (1 test)

nopol_examples.nopol_example_8.NopolExampleTest#test_11
.E
Time: 0.001
There was 1 failure:
1) initializationError(org.junit.runner.manipulation.Filter)
java.lang.Exception: No tests found matching Method test_11(nopol_examples.nopol_example_8.NopolExampleTest) from org.junit.internal.requests.ClassRequest@4830c68b
	at org.junit.internal.requests.FilterRequest.getRunner(FilterRequest.java:35)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:138)
	at fr.inria.lille.repair.MethodTestRunner.runTest(MethodTestRunner.java:44)
	at fr.inria.lille.repair.MethodTestRunner.access$000(MethodTestRunner.java:12)
	at fr.inria.lille.repair.MethodTestRunner$1.call(MethodTestRunner.java:20)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)

FAILURES!!!
Tests run: 1,  Failures: 1

nopol_examples.nopol_example_8.NopolExampleTest#test_10
.
Time: 0.005

OK (1 test)

nopol_examples.nopol_example_8.NopolExampleTest#test_9
.
Time: 0.003

OK (1 test)

nopol_examples.nopol_example_8.NopolExampleTest#test_5
.
Time: 0.002

OK (1 test)

nopol_examples.nopol_example_8.NopolExampleTest#test_6
.
Time: 0.004

OK (1 test)

nopol_examples.nopol_example_8.NopolExampleTest#test_7
.
Time: 0.002

OK (1 test)

nopol_examples.nopol_example_8.NopolExampleTest#test_8
.
Time: 0.002

OK (1 test)

